Runtime/Plataforms/AndroidBuildPlatform.cs

```diff
@@ -0,0 +1,121 @@
+            public string KeystorePassword;
+            public string AliasName;
+            public string AliasPassword;
+        }
+        [ConditionalField("UseKeystore")] public KeyStoreSettingsStruct KeyStoreSettings;
+
+        [MenuItem("Help/Automation/SetAndroidCredentials")]
+        public static void SetAndroidSignCredentials(BuildScript buildScript)
+        {
+            if(buildScript == null) return;
+            if(buildScript.TryGet(out AndroidBuildPlatform androidBuildPlatform))
+            {
+                if (!androidBuildPlatform.UseKeystore) return;
+                PlayerSettings.Android.useCustomKeystore = true;
+                PlayerSettings.Android.keystoreName = GetProjectPath() + androidBuildPlatform.KeyStoreSettings.KeystorePath;
+                PlayerSettings.Android.keystorePass = androidBuildPlatform.KeyStoreSettings.KeystorePassword;
+                PlayerSettings.Android.keyaliasName = androidBuildPlatform.KeyStoreSettings.AliasName;
+                PlayerSettings.Android.keyaliasPass = androidBuildPlatform.KeyStoreSettings.AliasPassword;
+            }
+            
+        }
+
+        public static void MakeDirectory(string relativePath)
+        {
+            var path = GetProjectPath() + relativePath;
+            if (Directory.Exists(path)) return;
+            Directory.CreateDirectory(path);
+        }
+
+        public override void OnBuild()
+        {
+            //PackageName
+            if (UsePackageName && !string.IsNullOrEmpty(PackageName))
+            {
+                PlayerSettings.SetApplicationIdentifier(BuildTargetGroup.Android, PackageName);
+            }
+
+            //Version
+            if (!UnityEditorInternal.InternalEditorUtility.isHumanControllingUs)
+            {
+                Version(out var gameVersion, out var bundleVersion);
+                PlayerSettings.bundleVersion = gameVersion;
+                PlayerSettings.Android.bundleVersionCode = bundleVersion;
+            }
+
+            //SetScriptDefinitions
+            if (UseCustomScriptDefinitions)
+            {
+                PlayerSettings.SetScriptingDefineSymbols(
+                    UnityEditor.Build.NamedBuildTarget.Android,
+                    ScriptDefinitions.Definitions
+                );
+            }
+
+            //Backend
+            PlayerSettings.SetScriptingBackend(BuildTargetGroup.Android, ScriptingBackend);
+
+            //Keystore Config
+            SetAndroidSignCredentials(buildScript);
+
+            if (BuildAAB)
+            {
+                //Make
+                const string subPathAab = "Build/Android/AAB/";
+                MakeDirectory(subPathAab);
+
+                var buildOptions = new BuildPlayerOptions
+                {
+                    scenes = GetScenes(),
+                    locationPathName = subPathAab + buildScript.GameName.ToLower().Replace(" ","_") + "_" + PlayerSettings.bundleVersion.Replace(".","_") + ".aab",
```

Comment:
breaking changes: new name will cause issues to people that are current using this package. Make this change a boolean option on this platform instead.